### DISCUSSION TITLE
get error ERR_FILE_NOT_FOUND on load fp.min.js

### DISCUSSION THREAD

**Discussion by fabriziodb** [fingerprintjs/fingerprintjs#630] [Q&A] [answered]
hi, i'm trying to make it work the fingerprint, but the html above give me err_file_not_found error. pasting the url in the browser i can read the content of the file. can anyone help me please?

```
<html>
 <head>
<script>
  function initFingerprintJS() {
    FingerprintJS.load().then(fp => {
      // The FingerprintJS agent is ready.
      // Get a visitor identifier when you'd like to.
      fp.get().then(result => {
        // This is the visitor identifier:
        const visitorId = result.visitorId;
        console.log(visitorId);
      });
    });
  }
</script>
<script
  async
  src="//cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs@3/dist/fp.min.js"
  onload="initFingerprintJS()"
></script>
</head>
<body>
</body>
</html>
```

**Comment by Valve** [comment, marked as answer]
@fabriziodb please try using full URL (with scheme) her: `https://cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs@3/dist/fp.min.js`

**Reply by fabriziodb** [reply]
ok, now it's working. also i need to add the username current logged on the page to the list of values used to calculate fingerprint, is there a way to do it in the new version?

can also u explain me the meaning of 31 in Fingerprint2.x64hash128?

thanks.

in version 2 i have this code:

```
Fingerprint2.get(function (components) {
                    try {

                        var values = components.map(function (component) { return component.value });
                        values.push(username);
                        var murmur = Fingerprint2.x64hash128(values.join(''), 31);

                        return murmur;

                    } catch (err) {

                    }
                });
```

**Reply by Finesse** [reply]
It’s possible, see https://github.com/fingerprintjs/fingerprintjs/blob/master/docs/migrating_v2_v3.md#customization

**Comment by Finesse** [comment]
What’s the URL of your HTML file? I guest it starts with `file:`. This explains why the relative address doesn’t work. In this case replace `//` with `https://` in the script tag.

**Reply by fabriziodb** [reply]
yes that was the problem thanks
